resources.py

```diff
@@ -337,5 +337,27 @@ def methylation_sites_mt_path(hail_version=CURRENT_HAIL_VERSION):
 context_mt_path = 'gs://gnomad-resources/constraint/context_processed.mt'
 
 
+# Sample QC files
+def qc_mt_path(data_type: str):
+    return 'gs://gnomad/sample_qc/mt/gnomad.{}.high_callrate_common_biallelic_snps.mt'.format(data_type)
+
+
+def qc_ht_path(data_type: str):
+    return 'gs://gnomad/sample_qc/ht/gnomad.{}.high_callrate_common_biallelic_snps.ht'.format(data_type)
+
+
+def qc_temp_data_path(data_type: str):
```

Comment:
This is more of a prefix than a path, no?

Reply:
It is. good call